src/main/java/de/hysky/skyblocker/compatibility/rei/SkyblockerREIClientPlugin.java

```diff
@@ -29,6 +42,51 @@ public void registerDisplays(DisplayRegistry displayRegistry) {
 
     @Override
     public void registerEntries(EntryRegistry entryRegistry) {
-        entryRegistry.addEntries(ItemRepository.getItemsStream().map(EntryStacks::of).toList());
+        List<EntryStack<ItemStack>> stacks = ItemRepository.getItemsStream()
+            .map(item -> {
+                EntryStack<ItemStack> entry = EntryStacks.of(item);
+                ClientEntryStacks.setTooltipProcessor(
+                    entry,
+                    (_entry, tooltip) -> modifyTooltip(
+                        entry.getValue(),
+                        tooltip.entries()
+                            .stream()
+                            .map(Tooltip.Entry::getAsText)
+                            .toList()
+                    )
+                );
+                return entry;
+            })
+            .toList();
+
+
+        entryRegistry.addEntries(stacks);
+    }
+
+    private final TooltipAdder[] adders = new TooltipAdder[]{
```

Comment:
It would be better to make the array inside of the tooltip manager public or add a getter rather than duplicate the array which will require people to duplicate new entries across different classes